docs/application/native/tutorials/details/sec-privileges.md

```diff
@@ -13,7 +13,7 @@ Tizen provides API-level access control for security-sensitive
 operations which, if not used correctly, can harm user privacy and
 system stability. Therefore, applications that use such sensitive APIs
 must declare the required privileges in the
-[tizen-manifest.xml](../process/setting-properties.md#setting-the-application-manifest) file.
+[`tizen-manifest.xml`](../process/setting-properties.md#setting-the-application-manifest) file.
```

Comment:
Please change 
[`tizen-manifest.xml`](../process/setting-properties.md#setting-the-application-manifest) to `tizen-manifest.xml`. 
Please don't provide path name for the file

Reply:
Actually, it is the hyperlink to the description about manifest. Are you sure to remove the link for it?

Reply:
@SunmLee, We donnot use hyperlink for tizen-manifest.xml file.

Reply:
I got it. It will be applied.

Reply:
Question. @sabeena21,  I think SunmLee's comment means, the link goes to the description "How to use the `tizen-manifest.xml` file", as the related information. 
If you check the link it jumps to the "Setting the Application Manifest" section of the setting-properties.md file, like below.

![image](https://user-images.githubusercontent.com/9958374/47005450-c2d2c200-d16e-11e8-9f8e-cb34c7512be7.png)

If the link to how to set the tizen-manifest.xml is to be removed, won't it better to leave that link at another place so that a user can refer?

Reply:
@SunmLee, please can you revert as [tizen-manifest.xml](../process/setting-properties.md#setting-the-application-manifest). Not required to add code tag for the hyperlink.

Reply:
@sabeena21 It was reverted at latest patch. Please confirm that.